server/documents/modules/rating.html.eco

```diff
@@ -251,6 +251,20 @@ themes      : ['Default']
       </div>
     </div>
 
+    <div class="example">
+      <h4 class="ui header">Without Javascript</h4>
+      <p>If you don't need user interaction, you can use the rating module as a pure readonly CSS element</p>
```

Comment:
I think 'readonly' should be 'read-only'.

Reply:
Fixed by https://github.com/fomantic/Fomantic-UI-Docs/pull/217/commits/cea0d25e21e26d1100f52de40b7c2aa86e646d06